.github/workflows/comment-pr-on-check-failure.yml

```diff
@@ -68,12 +50,48 @@ jobs:
         with:
           script: |
             var fs = require('fs');
-            var issue_number = Number(fs.readFileSync('./NR'));
-            const msg = fs.readFileSync('./msg',{encoding: 'utf8'})
+            const msg = fs.readFileSync('./msg',{encoding: 'utf8'});
+            var issue_number = ${{ github.event.workflow_run.pull_requests[0].number }};
+            var commentId = 0;
+            var oldMsg = '';
+            var updMsg = '';
+            var newMsg = '';
+            var shortCommitId = process.env.COMMIT_ID.substr(0,8);
+
+            const commentsList = await github.issues.listComments({
+                                        owner: context.repo.owner,
+                                        repo: context.repo.repo,
+                                        issue_number: issue_number,
+                                      });
+
+            for (const c of commentsList['data']) {
+              oldMsg = c.body;
+              if( oldMsg.includes('Oops! Looks like you failed the ') && oldMsg.includes(process.env.WORKFLOW_NAME) ) {
+                  commentId = c.id;
+                  if( process.env.WORKFLOW_STATUS == 'failure' )  {
+                      updMsg = ":recycle: Updated: :x: The check is still failing the " + process.env.WORKFLOW_NAME + " on last commit id: [" + shortCommitId + "](" + process.env.COMMIT_ID_URL + ")";
+                  }
+                  else  {
```

Comment:
here instead of else do 
`elsif (process.env.WORKFLOW_STATUS == 'success')`

Reply:
> here instead of else do `elsif (process.env.WORKFLOW_STATUS == 'success')`

@quentinDERORY - Made the code changes to take care of this comment as well.